helmfile/values/thanos/receiver.yaml.gotmpl

```
@@ -13,12 +13,16 @@ compactor:
   retentionResolution5m: {{ .Values.thanos.compactor.retentionResolution5m }}
   retentionResolution1h: {{ .Values.thanos.compactor.retentionResolution1h }}
 
-  {{- if eq .Values.thanos.compactor.deduplication "receiverReplicas" }}
+  {{- if eq .Values.thanos.compactor.verticalCompaction true }}
+  extraFlags:
+    - --compact.enable-vertical-compaction
+  {{- else if and (ne .Values.thanos.compactor.verticalCompaction true ) (or (eq .Values.thanos.compactor.deduplication "receiverReplicas") (eq .Values.thanos.compactor.deduplication "prometheusReplicas")) }}
   extraFlags:
+  {{- end }}
+  {{- if eq .Values.thanos.compactor.deduplication "receiverReplicas" }}
```

this is hard to read. can we instead have something like this:
```suggestion
  extraFlags:
  {{- if eq .Values.thanos.compactor.verticalCompaction true }}
    - --compact.enable-vertical-compaction
  {{- end }}
  {{- if eq .Values.thanos.compactor.deduplication "receiverReplicas" }}
```